### PR TITLE
Security and Annex B related IESG comments from Eric R

### PR DESCRIPTION
**Pull request by cdh4u**

### CODE REVIEW

File: draft-ietf-ice-rfc5245bis.xml [hunk 1]
```diff
@@ -4144,10 +4144,17 @@ subsections consider these attacks and their countermeasures.
...
-possible false conclusions an attacker can try and cause are:
+An attacker might attempt to disrupt the STUN connectivity checks.
+Ultimately, all of these attacks fool an ICE agent into thinking
+something incorrect about the results of the connectivity checks.
+Depending on the type of attack, the attacker needs to have different
+capabilities. In some cases the attacker needs to be on the path of the
+connectivity checks. In other cases the attacker does not need to be on
+the path, as long as it is able to generate STUN connectivity checks.
+While attacks on connectivity checks are typically performed by network
+entities, if an attacker is able to control an endpoint it can might be able
```
Comment (akeranen) [inline draft-ietf-ice-rfc5245bis.xml]: s/can might/might/

File: draft-ietf-ice-rfc5245bis.xml [hunk 2]
```diff
@@ -4227,13 +4235,14 @@ spoofed ICMP attacks is possible also for off-path attackers from a node
...
-fake success response. The attacker won't need to worry about
-disrupting the actual response since, if the candidate is not valid,
-it presumably wouldn't be received anyway. However, like the fake
-invalid attack, this attack is mitigated by the STUN short-term
-credential mechanism in conjunction with a secure candidate exchange.
+fake success response, or route (e.g., using a tunnelling mechanism) a
+success response that normally would be dropped or rejected by the
+network to the agent. Again, due to the STUN short-term credential
+mechanism, in order for the attacker to inject a valid success response, the
+attacker needs the password.
```
Comment (akeranen) [inline draft-ietf-ice-rfc5245bis.xml]: For the tunnelling attack no password is needed by the attacker (but this text seems to imply it's needed). To fix could move the last sentence before the "route attack" part and have the route attack part in the end.

**Review by akeranen — CHANGES_REQUESTED** [review]

File: draft-ietf-ice-rfc5245bis.xml (hunk 1 above)
Comment (cdh4u) [reply draft-ietf-ice-rfc5245bis.xml]: Will fix.

File: draft-ietf-ice-rfc5245bis.xml (hunk 2 above)
Comment (cdh4u) [reply draft-ietf-ice-rfc5245bis.xml]: Will fix.

**Pull request merged**